tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py

```diff
@@ -45,6 +46,7 @@
     Op.RETURN,
     Op.REVERT,
     Op.INVALID,
+    Op.RETF,
```

Comment:
This might be incorrect.

Reply:
the comment is incorrect lets remove it as it now used by other tests.
the meaning of the array if fine. this instructions stop code execution. retf  is like return

Reply:
But halting IMO should only be the ones that stop execution in the current contract context and returns it to the caller contract (or ends the tx). So with that in mind, JUMPF and RETF only change the current code section context but are not really halting the contract execution.

Reply:
ah, halting I meant stopping the code in the current context, not the global execution neceserraly

Reply:
Updated the test for better categorization.